advanced/Scripts/chronometer.sh

```diff
@@ -177,7 +224,7 @@ get_sys_stats() {
       ph_lte_ver="${ph_ver_raw[1]}"
       ph_ftl_ver="${ph_ver_raw[2]}"
     else
-      ph_core_ver="${COL_LIGHT_RED}API unavailable${COL_NC}"
+      ph_core_ver="${COL_LIGHT_RED}Offline${COL_NC}"
```

Comment:
I don't understand the logic why you show a red "Offline" when the version cannot be obtained.